Detectors/MUON/MCH/Calibration/src/BadChannelCalibrator.cxx

```diff
@@ -34,14 +34,12 @@ void BadChannelCalibrator::initOutput()
 bool BadChannelCalibrator::readyToSend(std::string& reason) const
 {
   reason = "";
-  if (BadChannelCalibratorParam::Instance().onlyAtEndOfStream) {
-    return false;
-  }
 
   // let's check our hypothesis about this object (nslots=1) is actually true
   auto nslots = getNSlots();
   if (nslots != 1) {
```

Comment:
I'm not sure we still need this check here as it is now duplicated in the `finalize` method which is anyways called, right ?

Reply:
I propose to keep it in order to make sure that the subsequent call to `hasEnoughData()` takes the expected slot as input. In any case this is a fatal condition that we should never encounter.